### PR TITLE
feat(eventbus): #CO-1701 add isInGarGroup action to event bus

### PR DESCRIPTION
**Pull request by Matiyeu**
## Describe your changes

Add isInGarGroup action to event bus. 
This action returns a list of structure ids that belong to the `RESP-AFFECT-GAR` group based on the structure and user.
\+ Implementation of CI and unit tests

The event bus has been added for the eLMS CAS on the entcore : [entcore#310](https://github.com/opendigitaleducation/entcore/pull/310)

## Issue ticket number and link

[#CO-1701](https://jira.support-ent.fr/browse/CO-1701)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/gar/service/impl/DefaultParameterService.java [hunk 1]
```diff
@@ -128,4 +130,23 @@ public void addUserToGarGroup(JsonObject body, Handler<Either<String, JsonObject
         }
         Neo4j.getInstance().execute(query, params, Neo4jResult.validUniqueResultHandler(handler));
     }
+
+    @Override
+    public void userHasGarGroup(JsonObject body, Handler<Either<String, JsonArray>> handler) {
```
Comment (JinDouang) [inline src/main/java/fr/openent/gar/service/impl/DefaultParameterService.java]: Pourquoi ne pas faire en promise ?

File: src/main/java/fr/openent/gar/service/impl/DefaultParameterService.java [hunk 2]
```diff
@@ -128,4 +130,23 @@ public void addUserToGarGroup(JsonObject body, Handler<Either<String, JsonObject
...
+        final String query = "MATCH (g:ManualGroup{name: {groupName} })<-[:IN]-(u:User{profiles:['Personnel']})--(s:Structure) " +
```
Comment (JinDouang) [inline src/main/java/fr/openent/gar/service/impl/DefaultParameterService.java:143]: A voir pour faire des TUs

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: .github/workflows/dev-check-repository.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      options: --user root -v ${{ github.workspace }}:/home/node/:rw
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run npm install
+        run: npm install
+      - name: Run build node with gulp
+        run: node_modules/gulp/bin/gulp.js build
+        #- name: Run test
+        #run: npm test
```
Comment (Matiyeu) [inline .github/workflows/dev-check-repository.yml:20]: The tests have not yet been implemented

File: src/main/java/fr/openent/gar/service/impl/DefaultParameterService.java [hunk 4]
```diff
@@ -128,4 +132,30 @@ public void addUserToGarGroup(JsonObject body, Handler<Either<String, JsonObject
...
+                "RETURN s.id AS structureId";
+        final JsonObject params = new JsonObject()
+                .put(Field.USER_ID, body.getString(Field.USER_ID))
+                .put(Field.STRUCTURE_IDS, body.getJsonArray(Field.STRUCTURE_IDS))
+                .put(Field.GROUP_NAME, GAR_GROUP_NAME)
+                .put("direction", FUNCTION_DIRECTION_NAME)
+                .put("documentation", FUNCTION_DOCUMENTATION_NAME);
+
+        Neo4j.getInstance().execute(query, params, Neo4jResult.validResultHandler(either -> {
+            if (either.isLeft()) promise.fail(either.left().getValue());
```
Comment (JinDouang) [inline src/main/java/fr/openent/gar/service/impl/DefaultParameterService.java]: log erreur

**Review by JinDouang — CHANGES_REQUESTED** [review]

**Pull request merged**